crates/core/src/key/database/ti.rs

```diff
@@ -10,7 +11,7 @@ use serde::{Deserialize, Serialize};
 pub struct Ti {
 	__: u8,
 	_a: u8,
-	pub ns: u32,
+	pub ns: NamespaceId,
```

Comment:
`NamespaceId` is marked as `#[repr(transparent)]` so this continues to work as expected. See the tests below for evidence.